float64 f64_cmd
uint16 ui16_cmd
bool enable
bool clear
bool ignore

